### PR TITLE
Added mentor and Geolocation (@gabru-md)

### PR DESCRIPTION
**Pull request by saarthakchats**

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -168,3 +168,9 @@
   image: vibhor.jpg
   lat: 28.613939
   lng: 77.209021
+
+- name: Manish Devgan
+  github: https://github.com/gabru-md
+  image: piyush.jpg
```
Comment (abishekvashok) [inline _data/mentors.yml:174]: Properly name the image as gabru-md or manish

**Review by abishekvashok — COMMENTED** [review]
It looks ok but it will be better if u could do the changes.

**Comment by saarthakchats** [comment]
@abishekvashok 
I am making a new pull request.

**Comment by abishekvashok** [comment]
Sure :)

**Pull request closed without merging**